00-programming-fundamentals/git-merge-conflicts.md

```diff
@@ -50,9 +52,14 @@ our version
 their version
 >>>>>>> c157db99220f47abfcb63547bfb8765ba571e19d
 ```
+With Atom highlighting:
+![atom merge conflict](images/merge-conflict1.png)
+
 
 The conflict markers are the lines that start with `<<<<<<<`, `=======`, or `>>>>>>>`. All of the lines between the left-pointing brackets and the equals signs are the **ours** version, and all of the lines between the equals signs and the right-facing brackets are the **theirs** version.
 
+Atom provides buttons to select the version you want to keep and this can be useful, but often you will need to edit the text manually and remove the git markers (>>>>, ==== and <<<<) manually.
```

Comment:
Maybe some slight wordsmithing on this:
* Remove either of the 'manually'
* I would probably say 'you may need' instead of 'often you will need', unless we feel pretty confident that Atom's merge conflict plugin is frequently not sufficient.